check/configuration/supported/phpstan.md

```diff
@@ -0,0 +1,47 @@
+---
+description: phpstan is a linter for PHP
+title: Trunk | How to run phpstan
+layout:
+  title:
+    visible: true
+  description:
+    visible: false
+  tableOfContents:
+    visible: true
+  outline:
+    visible: true
+  pagination:
+    visible: true
+---
+
+# phpstan
```

Comment:
@laurit17 can we get some top level docs drafted for PHP this week (probs should also be called out on the individual linter pages too)?